### PR TITLE
Executive Test Traceability Card Implementation

### PR DESCRIPTION
**Pull request by nameisaravind**
This card is created to implement traceability card in executive and this PR has to be merged after https://github.com/Hygieia/Hygieia/pull/2880

### CODE REVIEW

**Comment by nameisaravind** [comment]
> I am not sure this is the correct way to go about this change. We should discuss this further.

@Sbrenthughes Traceability card implementation is updated as per the discussion had to map it through collectorItemId. This PR has to be merged after this PR has to be merged after https://github.com/Hygieia/Hygieia/pull/2880

**Comment by rvema** [comment]
@nameisaravind please resolve the conflicts and update so that the change can be accepted

**Comment by Courtneyp123** [comment]
@nameisaravind Hello! Can you resolve the conflicts for this PR?

**Comment by nameisaravind** [comment]
@Courtneyp123 @rvema resolved conflicts

**Comment by rvema** [comment]
@nameisaravind Can you resolve the conflicts for this PR so that the same can be merged?

**Comment by rvema** [comment]
@nameisaravind Can you fix the compilation error in this PR?

**Comment by nameisaravind** [comment]
@rvema could you check now...

File: exec-ui/src/app/modules/metrics/modules/shared/components/metric-preview/metric-preview-base.component.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
       <div class="key-indicators" *ngIf="this.hasReports()">
         <div class="secondary-metric" *ngFor="let metric of this.metricPreview.secondaryMetrics">
           <p>{{metric.name}}</p>
-          <div>{{metric.value}} {{metric.unit}}</div>
+          <div>{{metric.value}}{{metric.unit}}</div>
```
Comment (rvema) [inline exec-ui/src/app/modules/metrics/modules/shared/components/metric-preview/metric-preview-base.component.html]: space should be present right?

Comment (nameisaravind) [reply exec-ui/src/app/modules/metrics/modules/shared/components/metric-preview/metric-preview-base.component.html]: no, it doesn't have to. @rvema

**Comment by rvema** [comment]
@nameisaravind please sign the cla - you might have received an email

**Pull request merged**